cli/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# 1. Edit DIRECTORY variable where the language packs are installed. Be sure you are on the branch you want to be.
+DIRECTORY='/Users/user/git/moodle-langpacks'
+
+# 2. Edit the branch name and USERINFO.
+USERINFO='Initial install <install@moodle.invalid>'
+
+# 3. Run the script from local/amos directory.
+# ./cli/install.sh
+
+# 4. Execute this SQL (number 2 is the userid to make amos manager).
+# Also an AMOS manager role should be added and assigned to the user.
+# INSERT INTO mdl_amos_translators ( userid, lang, status ) VALUES (2, 'X', 0);
+
+pushd "$DIRECTORY" > /dev/null
+BRANCH=`git branch 2>/dev/null | grep '*' | sed 's/* \(.*\)/\1/'`
```

Comment:
Please note that it is not recommended to rely on the particular output syntax in Git. The better and reliable machine-readable way to get the currently checked out branch is e.g.
```
git symbolic-ref --short HEAD
```